coop_traj_pid:
  ros__parameters:
    # Base 1 PID parameters
    base1_pid_x_kp: 1.0
    base1_pid_x_ki: 0.0
    base1_pid_x_kd: 0.0

    base1_pid_y_kp: 1.0
    base1_pid_y_ki: 0.0
    base1_pid_y_kd: 0.0

    base1_pid_theta_kp: 1.0
    base1_pid_theta_ki: 0.0
    base1_pid_theta_kd: 0.0

    # Base 2 PID parameters
    base2_pid_x_kp: 1.0
    base2_pid_x_ki: 0.0
    base2_pid_x_kd: 0.0

    base2_pid_y_kp: 1.0
    base2_pid_y_ki: 0.0
    base2_pid_y_kd: 0.0

    base2_pid_theta_kp: 1.0
    base2_pid_theta_ki: 0.0
    base2_pid_theta_kd: 0.0

    # Control loop time step (in seconds)
    dt: 0.01
  